src/arch/riscv64/CodeGen.zig

```diff
@@ -1159,6 +1161,12 @@ fn airPopcount(self: *Self, inst: Air.Inst.Index) !void {
     return self.finishAir(inst, result, .{ ty_op.operand, .none, .none });
 }
 
+fn airUnaryMath(self: *Self, inst: Air.Inst.Index) !void {
+    const un_op = self.air.instructions.items(.data)[inst].un_op;
+    const result: MCValue = if (self.liveness.isUnused(inst)) .dead else return self.fail("TODO implement airUnaryMath for {}", .{self.target.cpu.arch});
```

Comment:
Could you please re-format this into multiple lines like you did for `x86_64` backend?

Reply:
I made this function match whatever the other ones were like for the respective backends. Would you like me to re-format anyway?

Reply:
Yes please.